src/components/Typography/Link.tsx

```diff
@@ -28,14 +28,26 @@ function Link({
     classes.push(className)
   }
 
+  function keyDownHandler(e: React.KeyboardEvent<HTMLAnchorElement>) {
```

Comment:
I think we are mis-using `<a>` tags here - can we take this out of this PR for now?

Reply:
@MildTomato done! Thanks for reviewing this 🙏

Reply:
👍 🚀